test/WebSites/BasicWebSite/Controllers/ApplicationBaseController.cs

```diff
@@ -1,10 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-namespace BasicWebSite.Controllers
-{
-    public abstract class ApplicationBaseController
```

Comment:
I did that at first, but I don't think that test is proving anything that isn't being tested in unit tests, so I think it can be safely removed.